CHANGELOG.md

```diff
@@ -3,6 +3,123 @@
+
+  ```diff
+  -params.rowIndex
+  -params.colIndex
+  +apiRef.current.getRowIndex(params.id)
+  +apiRef.current.getColumnIndex(params.field)
+  ```
+
+- [DataGrid] Calling `params.getValue` now requires the id to be passed (#1513) @dtassone
+
+  ```diff
+  -params.getValue(field)
+  +params.getValue(params.id, field)
+  ```
+
+- [DataGrid] Rename CSS classes (#1660) @DanailH
+
+  1. `MuiDataGrid-colCellWrapper` to `MuiDataGrid-columnHeaderWrapper`
+  2. `MuiDataGrid-colCell` to `MuiDataGrid-columnHeader`
+  3. `MuiDataGrid-colCellCheckbox` to `MuiDataGrid-columnHeaderCheckbox`
+  4. `MuiDataGrid-colCellSortable` to `MuiDataGrid-columnHeaderSortable`
+  5. `MuiDataGrid-colCellCenter` to `MuiDataGrid-columnHeaderCenter`
+  6. `MuiDataGrid-colCellLeft` to `MuiDataGrid-columnHeaderLeft`
+  7. `MuiDataGrid-colCellRight` to `MuiDataGrid-columnHeaderRight`
+
+- [XGrid] Calling `setCellFocus` now requires the id and field to be passed (#1513) @dtassone
+
+  ```diff
+  -apiRef.current.setCellFocus: (indexes: GridCellIndexCoordinates) => void;
+  +apiRef.current.setCellFocus: (id: GridRowId, field: string) => void;
+  ```
+
+- [XGrid] Rename `apiRef` methods (#1513) @dtassone
+
+  Changes on `apiRef.current`:
+
+  ```diff
+  -apiRef.current.getRowIndexFromId: (id: GridRowId) => number;
+  +apiRef.current.getRowIndex: (id: GridRowId) => number;
+  ```
+
+- [XGrid] Rename `apiRef` methods (#1667) @m4theushw
+
+  Changes on `apiRef.current`:
+
+  ```diff
+  -apiRef.current.getColumnFromField: (field: string) => GridColDef;
+  -apiRef.current.getRowFromId: (id: GridRowId) => GridRowModel;
+  +apiRef.current.getColumn: (field: string) => GridColDef;
+  +apiRef.current.getRow: (id: GridRowId) => GridRowModel;
+  ```
+
+#### Changes
+
+- [DataGrid] Add Slovak (skSK) locale (#1634) @martinvysnovsky
+- [DataGrid] Add `columnHeader`, `row` and `cell` in addition to `root` in classes prop (#1660) @DanailH
+- [DataGrid] Add `isRowSelectable` prop (#1659) @m4theushw
+- [DataGrid] Add sort icon for when column is unsorted (#1415) @m4theushw
+- [DataGrid] Fix `id` and `aria-labelledby` attributes on the column menu (#1460) @m4theushw
+- [DataGrid] Fix broken checkbox in Material-UI v5 (#1587) @ZeeshanTamboli
+- [DataGrid] Fix CSS classes prefix (#1693) @m4theushw
+- [DataGrid] Fix German (deDe) locale (#1624) @klinge27
+- [DataGrid] Fix filter with object as value and value getter (#1665) @dtassone
+- [DataGrid] Fix incorrect date selection (#1652) @aTmb405
+- [DataGrid] Fix overflow of maximum page (#1583) @oliviertassinari
+- [DataGrid] Fix typo in Italian (itIT) locale (#1635) @profcav
+- [DataGrid] Improve performance of width resizing (#1686) @dtassone
+- [DataGrid] Make rows immutable for better developer experience (#1661) @ZeeshanTamboli
+- [DataGrid] Pass state values as props (#1628) @m4theushw
+- [DataGrid] Improve performance with filtering, sorting, and rendering (#1513) @dtassone
+- [XGrid] Fix checkbox column resizing (#1682) @elyesbenabdelkader
```

Comment:
I don't think we've prefixed something with `XGrid` before but maybe we should start? Just a note.

Reply:
We use [XGrid] when the feature is only available in the commercial version and [DataGrid] when it's also available in the MIT version. We started doing it about 6 months ago.